### PR TITLE
Create a menu in the Windows Start Menu

### PR DESCRIPTION
**Pull request by alixdamman**
You can test the package by running:
$conda install -c larray-test larray-editor

### CODE REVIEW

File: condarecipe/larray-editor/larray-editor.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            {
+                "name": "LArray Editor",
+                "pywscript": "${PYTHON_SCRIPTS}/larray-editor-script.pyw",
+                "icon": "${MENU_DIR}/larray.ico",
+                "quicklaunch": true
+            },
+            {
+                "name": "LArray Documentation",
+                "webbrowser": "http://larray.readthedocs.io/en/stable",
+                "icon": "${MENU_DIR}/larray-help.ico"
```
Comment (gdementen) [inline condarecipe/larray-editor/larray-editor.json]: where does this icon come from? Do we have the rights to use it? Is it in the public domain?

**Comment by alixdamman** [comment]
file larray-help.ico updated (see commit message)

**Comment by gdementen** [comment]
LGTM

**Comment by alixdamman** [comment]
Thanks for review

**Pull request merged**